internal/fullnode/service_builder.go

```diff
@@ -65,12 +70,51 @@ func BuildServices(crd *cosmosv1.CosmosFullNode) []diff.Resource[*corev1.Service
 			svc.Spec.Type = corev1.ServiceTypeClusterIP
 		}
 
-		p2ps[i] = diff.Adapt(&svc, i)
+		svcs[i] = diff.Adapt(&svc, i)
 	}
 
 	rpc := rpcService(crd)
 
-	return append(p2ps, diff.Adapt(rpc, len(p2ps)))
+	if crd.Spec.Type == cosmosv1.Sentry {
+		for i := int32(0); i < crd.Spec.Replicas; i++ {
+			ordinal := i
+			var svc corev1.Service
+			svc.Name = sentryServiceName(crd, ordinal)
+			svc.Namespace = crd.Namespace
+			svc.Kind = "Service"
+			svc.APIVersion = "v1"
+
+			svc.Labels = defaultLabels(crd,
+				kube.InstanceLabel, instanceName(crd, ordinal),
+				kube.ComponentLabel, "sentry",
```

Comment:
```suggestion
				kube.ComponentLabel, "cosmos-sentry",
```
according to https://github.com/strangelove-ventures/horcrux-proxy/blob/bc8aec6c4b82c8162e117ddcad3f3a01532f3478/cmd/watcher.go#L22